yesod-test/test/main.hs

```diff
@@ -575,9 +580,55 @@ main = hspec $ do
             statusIs 200
             (requireJSONResponse :: YesodExample site [Text]) `liftedShouldThrow` (\(e :: SomeException) -> True)
 
+    describe "Checks the order of addPostParam arguments addition" $ yesodSpec defaultRoutedApp $ do
+        yit "Creates a form with a tupleField" $ do
+            get DisplayR
+
+            request $ do
+              addTokenFromCookie
+              setUrl TupleR
+              setMethod "POST"
+              addPostParam "data" "ABC"
+              addPostParam "data" "ABC"
+            statusIs 200
```

Comment:
Why would this be 200? We should not be able to parse an `(Int, Text)` from `[ "ABC", "ABC" ]`, since we can't parse an `Int` from `"ABC"`. How will you represent failure here?

Reply:
It is now 400 as it should be.